scripts/schmidt_comparison.py

```diff
@@ -0,0 +1,48 @@
+import sys
+sys.path.insert(0,'../chamber/models')
+
+from CoolProp import HumidAirProp as hap
+import pandas as pd
+
+import props
+
+
+results = {'Pressure (Pa)': [], 'Temperature (K)': [],
+					'Ralative Humidity': [], 'Schmidt Mills': [],
+					'Schmidt Marrero': []}
+
+
+# Gets relevant values and calculates Schmidt number
+def get_schmidt(p, t, rh):
+	mills = props.get_d_12(p, t, 'Mills')
+	marrero = props.get_d_12(p, t, 'Marrero')
+
+	# Sometimes errors arise with coolprop from high relative humidities
```

Comment:
What are these errors? We should put something in place to catch them. Can you share what the error message is attached to the `ValueError`? 

*Note*: Relative humidity should be a fraction between 0 and 1, not a percentage between 0 and 100. The loop on line `37` looks like it is going from 10 to 80 rather than 0.1 to 0.8. If you fix this, you can probably get rid of the `try: except:` statement.

Reply:
That makes much more sense, thanks!

Reply:
Glad we were able to identify the error. Thanks for your help Alex!